test/IDE/complete_unresolved_members.swift

```diff
@@ -148,7 +148,7 @@ class C4 {
 // UNRESOLVED_3-DAG: Decl[InstanceMethod]/CurrNominal/TypeRelation[Invalid]: hash({#(self): SomeEnum1#})[#(into: inout Hasher) -> Void#]; name=hash(:)
 
 // Exhaustive to make sure we don't include `init({#(some):` or `init({#nilLiteral:` entries
-// UNRESOLVED_3_OPT: Begin completions, 9 items
+// UNRESOLVED_3_OPT: Begin completions, 10 items
```

Comment:
Could you add the new items to the `-DAG` lines below? Same for the test in complete_generic_optional.swift.

Reply:
D'oh, good catch, thanks! I thought I got them all.

Reply:
I added the missing item to this case, but I left complete_generic_optional.swift as is -- it is not intended to be an exhaustive test.